src/hdmf/term_set.py

```diff
@@ -94,3 +115,39 @@ def __getitem__(self, term):
         except KeyError:
             msg = 'Term not in schema'
             raise ValueError(msg)
+
+    def schemasheets_convert(self):
+        try:
+            import yaml
+            from linkml_runtime.utils.schema_as_dict import schema_as_dict
+            from schemasheets.schemamaker import SchemaMaker
+        except ImportError:
+            msg="Install schemasheets."
+            raise ValueError(msg)
+        schema_maker = SchemaMaker()
+        tsv_file_paths = glob.glob(self.schemasheets_folder + "/*.tsv")
+        schema = schema_maker.create_schema(tsv_file_paths)
+        schema_dict = schema_as_dict(schema)
+        schemasheet_schema_path = os.path.join(self.schemasheets_folder, f"{schema_dict['name']}.yaml")
+
+        with open(schemasheet_schema_path, "w") as f:
+            yaml.dump(schema_dict, f)
+
+        return schemasheet_schema_path
+
+    def enum_expander(self):
```

Comment:
Pleas add a docstring for ``enum_expander`` to indicate what the function does.

Reply:
If I understand the function correctly, then this is creating a new file with the expanded version of the dynamic enumeration and places it next to the original term set. If so then maybe calling the function ``expand_term_set`` would be more intuitive.

Reply:
Looking at the tutorial it seems that users are not supposed to call this function directly, but are instead supposed to construct ``TermSet(...., dynamic=True)``. If that is the case, then I would add  the `__` prefix to the function name to make it private.

Reply:
Will do.